comps/prompt_registry/mongo/README.md

```diff
@@ -50,7 +50,7 @@ The Prompt Registry microservice exposes the following API endpoints:
 
   ```bash
   curl -X 'POST' \
-    http://{host_ip}:6012/v1/prompt/create \
```

Comment:
s/{host_ip}/${host_ip}/
The same for all {host_ip} lines to reference an environmental var.